modules/oe_theme_content_consultation/oe_theme_content_consultation.module

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming the Consultation content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\oe_content\CallEntityWrapperInterface;
+use Drupal\oe_content_consultation\ConsultationNodeWrapper;
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function oe_theme_content_consultation_node_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  if ($entity->bundle() !== 'oe_consultation') {
+    return;
+  }
+
+  if ($build['#view_mode'] === 'full') {
+    // Override field labels.
+    if ($entity->get('oe_departments')->count() === 1) {
+      $build['oe_departments']['#title'] = t('Department');
+    }
+
+    // Show respond to the consultation information based on open status.
```

Comment:
There is a rule in the spec https://webgate.ec.europa.eu/fpfis/wikis/display/EWPP/Consultation+Content+Type+Specification+-+Version+1.0.0 

> Respond to the consultation (closed status text): This is displayed if the status is  Closed. If nothing is filled in then display this: "The response period for this consultation has ended. Thank you for your input."